CenterForOpenScience/styles
# Update citation formats we support.

Ticket: https://openscience.atlassian.net/browse/OSF-8515

## QA Notes:

It is worth to do another pass over all the citation formats to see what's not working since we add 400+ more citation formats in this PR.
